tasks/pip.yml

```diff
@@ -328,3 +328,18 @@
   register: result
   until: result is success
   notify: reload supervisor
+
+- name: Install django
+  pip:
+    name: "django{{ openwisp2_django_version}}"
```

Comment:
In the current state, this line will evaluate to `django3.2.0` which is wrong.

Reply:
Okay, I understood. I am improving this. Thanks

Reply:
```suggestion
    name: "django~={{ openwisp2_django_version}}"
```